Cargo.toml

```diff
@@ -13,7 +13,7 @@ license = "MIT OR Apache-2.0"
 clap = { version = "^4.3", features = ["derive"] }
 fluent-uri = "0.1.4"
 miette = "7.5.0"
-nickel-lang-core = { version = "0.10", default-features = false }
+nickel-lang-core = { version = "0.12", default-features = false }
```

Comment:
```suggestion
nickel-lang-core = { version = "0.13", default-features = false }
```

Reply:
nickel-lang-core isn't in sync with the language version, in fact there's a diff of one. So `0.13` corresponds to Nickel `1.12`.